test/server/utils/findPort.test.js

```diff
@@ -2,7 +2,7 @@
 
 const http = require('http');
 const portfinder = require('portfinder');
-const findPort = require('../../../lib/utils/findPort');
+const findPort = require('../../../lib/Server').findPort;
```

Comment:
Ideally we should move this test to `Server.test.js`

Reply:
no problem here too, isolated test

Reply:
Yes, move it to `Server.test.js`